site/app/templates/admin/RainbowCustomization.twig

```diff
@@ -307,7 +366,10 @@ To remove a curve simply blank all boxes."></i>
             $(`#override-${gradeable_id}`).html(override.val());
         });
 
-        $("#user_id").autocomplete({
+        $("#plagiarism_user_id").autocomplete({
+            source: {{ student_full|raw }}
+        });
+        $("#manual_grading_user_id").autocomplete({
```

Comment:
["#plagiarism_user_id", "#manual_grading_user_id"].forEach(function(id) {
    $(id).autocomplete({
        source: {{ student_full|raw }}
    });
});

make it concise

Reply:
[f493878](https://github.com/Submitty/Submitty/pull/10653/commits/f4938789c799c3f17fa6f4ef4d1e514acd043624)